src/platforms/platform.dom.js

```diff
@@ -141,38 +92,100 @@ module.exports = function(Chart) {
 		return canvas;
 	}
 
-	return {
-		/**
-		 * Creates a Chart.js event from a raw event
-		 * @method BrowserPlatform#createEvent
-		 * @implements IPlatform.createEvent
-		 * @param e {Event} the raw event (such as a mouse event)
-		 * @param chart {Chart} the chart to use
-		 * @returns {Core.Event} the chart.js event for this event
-		 */
-		createEvent: function(e, chart) {
-			var relativePosition = helpers.getRelativePosition(e, chart);
-			return {
-				// allow access to the native event
-				native: e,
-
-				// our interal event type
-				type: typeMap[e.type],
-
-				// width and height of chart
-				width: chart.width,
-				height: chart.height,
-
-				// Position relative to the canvas
-				x: relativePosition.x,
-				y: relativePosition.y
-			};
-		},
+	function createEvent(chart, type, x, y, native) {
+		var canvas = chart.canvas;
+		return {
+			type: type,
+			chart: chart,
+			native: native || null,
+			width: readUsedSize(canvas, 'width'),
```

Comment:
does this change the behaviour to be the CSS width? I think the old width and height were the render width and height (ie. including any scaling because of the devicePixelRatio)